### DISCUSSION TITLE
Mastodon API calls intermittedly returing 400

### DISCUSSION THREAD

**Discussion by pitosalas** [mastodon/mastodon#30519] [General]
I have an IFTTT action triggered by raindrop.io to post to mastodon. It returns 400 errors, what looks to me, intermittedly. I believe the format is correct, as I've tried and tested it many times. Would a. non-ascii "smartquote" in the payload be a cause for such an error?

**Comment by mjankowski** [comment]
Is this still happening? Can you post a replicable request/response that shows the issue?

**Comment by pitosalas** [comment]
No. Ok to close. Thanks.

Best,

Pito

Boston Robot Hackers && 
Comp. Sci Faculty, Brandeis University (Emeritus)
